node/Cargo.toml

```
@@ -1,24 +1,31 @@
 [package]
 name = "qf-node"
-description = "Quantum Fusion solochain node"
+description = "A parachain node built with Substrate and Cumulus. From commit 3793fc3444e28d309e67513fda5afe153a2aa928 at https://github.com/QuantumFusion-network/polkadot-sdk"
```

Please move version mentioning to a code comment or readme.

```suggestion
Quantum Fusion Network Client Node.
```